flake8_spellcheck/__init__.py

```diff
@@ -129,17 +129,21 @@ def _detect_errors(self, tokens, use_symbols, token_type):
         code = get_code(token_type)
 
         for pos, token in tokens:
+            value = token
```

Comment:
why perform the rename here? Is it because you are later mutating that value?

Reply:
Yes, only in some conditional code paths. Then, the common code can simply refer to `value`.